.github/workflows/bindings_python.yml

```diff
@@ -37,11 +41,22 @@ jobs:
           working-directory: "bindings/python"
           command: build
           args: --release --sdist -o dist --find-interpreter
+          sccache: "true"
```

Comment:
Sadly, we can't enable sccache because we need to run `ghac`'s integration tests. Enabling sccache will lead to our tests always failed for rate limit.